generics/services/google-cloud.js

```diff
@@ -70,7 +70,60 @@ let getDownloadableUrl = ( filename,bucketName) =>{
   })
 }
 
+/**
+ * Get google cloud signed url.
+ * @method
+ * @name signedUrl
+ * @param {String} [folderPath = ""] - link to the folder path.
+ * @param {Array} [fileName = ""] - name of the file.
+ * @param {Array} bucketName - name of the bucket.  
+ * @returns {Object} - signed url and gcp file name. 
+ */
+
+let signedUrl = ( folderPath = "", fileName = "", bucketName ) => {
+  return new Promise(async (resolve, reject) => {
+    try {
+
+      var myBucket = storage.bucket(bucketName);
+      
+      if(folderPath == "" || fileName == "") {
+        throw new Error(httpStatusCode.bad_request.status);
+      }
+      
+      let noOfMinutes = 30;
```

Comment:
@aman-tunerlabs - Can this expiry no of minutes be a common constant across services